docs/codeql/reusables/vs-code-basic-instructions/run-quick-query-2.rst

```diff
@@ -0,0 +1,7 @@
+4. Save the query in its default location (a temporary "Quick Queries" directory under the workspace for ``GitHub.vscode-codeql/quick-queries``).
+
+#. Right-click in the query window and select **CodeQL: Run Query**. (Alternatively, run the command from the Command Palette.)
+
+   The query will take a few moments to return results. When the query completes, the results are displayed in a CodeQL Query Results window, alongside the query window.
```

Comment:
I am not sure if this established VSCode terminology, but I would refer to these as tabs or views rather than windows. Perhaps something like this:
```suggestion
   The query will take a few moments to return results. When the query completes, the results are displayed in a CodeQL Query Results view, next to the main editor view.
```
I also changed:
- "alongside" to "next to", since "alongside" could be misinterpreted to mean that the "query window" shows up once the query is run as well. 
- "query window" to "main editor view", since "query window" might make someone expect something specific to queries.

Reply:
Thanks for suggestion. I'll update the other articles to follow suit.